packages/react-html/CHANGELOG.md

```diff
@@ -5,7 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+### Changed
+
+- The `<Script />` and `<Style />` components are now exported from `@shopify/react-html/server` ([#1648](https://github.com/Shopify/quilt/pull/1648))
```

Comment:
I understand why you would want to do this (to be more consistent with Link I presume) but this is actually going to be a breaking change since the<link/> that is meant to be inline will now to to the head and the css specificity will change.

Reply:
Yep, it's a breaking change but for the good of the library. Consumers should not use `Style` or `Script` directly.

Reply:
my worry is that this will be too easily missed when consumer upgrade to the next version of react-html.

I think we should at least add a carefully documented migration guide and also mentioned in the README how the exported version of <Style /> component had changed.

Reply:
I will add the migration doc for sure. The API of the component change so Typescript will complain also.

Reply:
Here is the new migration steps: https://github.com/Shopify/quilt/pull/1648/files#diff-2c1ecb54d6ceef3255296a43f4ddf9c8